data_Insights_website/templates/pages/docs&help.html

```diff
@@ -0,0 +1,141 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="../../static/css/bootstrap.min.css">
+    <!-- Hover CSS-->
+    <link rel="stylesheet" href="../../static/css/hover-min.css">
+    <!-- Website's CSS-->
+
+    <link rel="stylesheet" href="../../static/css/style.css">
+    <!-- Particles.js -->
+    <script src="../../static/js/particles.min.js"></script>
+    <!-- Fonts js-->
+    <script src="https://use.edgefonts.net/rationale;quicksand.js"></script>
+
+    <script src="../../static/js/typed.min.js"></script>
+
+    <title> </title>
+</head>
+
+<body>
+
+    <!-- Particles background -->
+    <script>
+        particlesJS.load('particles-bg', 'particles-configs/particlesjs-config.json', function() {
+            console.log('callback - particles.js config loaded');
+        });
+
+    </script>
+    <div id="particles-bg"></div>
+
+    <!-- Page content-->
+    <div class="container-fluid" id="page-content">
+
+        <!--Navigation bar-->
+        <div class="navbar navbar-expand-sm sticky-top">
+            <!--Logo-->
+            <a href="#" class="navbar-brand">
+                <img src="../../static/img/logo.png" alt="">
+            </a>
+
+            <button class="navbar-toggler" type='button' data-toggle='collapse' data-target="#nav">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <!--Navbar items-->
+            <div class="collapse navbar-collapse" id="menu">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item">
+                        <a href="#" class="nav-link hvr-overline-from-center active">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#about" class="nav-link hvr-overline-from-center">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#get-involved" class="nav-link hvr-overline-from-center">Get Involved</a>
```

Comment:
elements of the navbar that are not on this page should be removed

Reply:
Done